microsite/src/pages/plugins/_pluginCard.tsx

```diff
@@ -9,14 +9,14 @@ export interface IPluginData {
   category: string;
   description: string;
   documentation: string;
-  iconUrl: string;
+  iconUrl?: string;
   title: string;
   addedDate: string;
   isNew: boolean;
   order?: number;
 }
 
-const defaultIconUrl = 'img/logo-gradient-on-dark.svg';
```

Comment:
The old works for https://backstage.io/plugins but not https://backstage.io/plugins/

I recommend to use the / here because all iconUrls uses the same prefix and they should be at least similar.